server/app/services/question/types/MultiOptionQuestionDefinitionConfig.java

```diff
@@ -0,0 +1,94 @@
+package services.question.types;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.OptionalLong;
+import services.LocalizedStrings;
+import services.question.QuestionOption;
+import services.question.types.MultiOptionQuestionDefinition.MultiOptionValidationPredicates;
+
```

Comment:
**suggestion:** Could you add a javadoc?

Reply:
Done.